packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -343,6 +338,6 @@ def _mint_objects(
                 mint_quantity=parameters.money_endowment,
                 ledger_api=self.context.ledger_apis.apis.get("ethereum"),
                 skill_callback_id=self.context.skill_id,
-                token_id=self.context.shared_state["token_ids"][0],
+                token_id=token_id,
```

Comment:
where is this defined?

Reply:
The token_id is a parameter in the function. None by default. If we want to mint a single token we need to provide the. token id in the function.